.devcontainer/socket_proxy.py

```
@@ -0,0 +1,90 @@
+        except Exception:
+            pass
+        try:
+            destination.close()
+        except Exception:
+            pass
+
+def handle_client(client_sock, target_path):
+    try:
+        target_sock = socket.socket(socket.AF_UNIX, socket.SOCK_STREAM)
+        target_sock.connect(target_path)
+    except Exception as e:
```

Ruff (0.15.18)

[warning] Do not catch blind exception: `Exception`

(BLE001)